apps/web/app/layout.tsx

```diff
@@ -29,10 +31,12 @@ export default function RootLayout({
   return (
     <html lang="ko">
       <body className={spoqaHanSansNeo.variable}>
+        <MobileSizeWatcher />
         <Gnb />
         <Providers>
           <div className="md:ml-200">{children}</div>
         </Providers>
+        <Toast />
```

Comment:
```suggestion
        <Providers>
           <Gnb />
          <div className="md:ml-200">{children}</div>
        </Providers>
```
MobileSizeWatcher랑 Toast는 providers에 구성되어도 좋을거같아 gnb는 providers children으로 들어가고

Reply:
외부 라이브러리 등등은 providers에서 관리하는 너낌?